workers/scripts/mapping_saturation.py

```diff
@@ -0,0 +1,57 @@
+"""Run uvicorn with custom options to start API server for development purposes"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ohsome_quality_analyst.indicators.mapping_saturation.fit import run_all_models
+from tests.unittests.mapping_saturation.fixtures import VALUES
+
+XDATA = np.asarray(range(len(VALUES)))
+YDATA = VALUES
+
+
+def run():
+    fits = run_all_models(XDATA, YDATA)
+
+    fig, axs = plt.subplots(2, 3, sharex=True, sharey=True)
```

Comment:
I often use something like this:
```
import matplotlib.pyplot as plt
import matplotlib.gridspec as gridspec

fig = plt.figure(figsize=(12.5, 6))
gs1 = gridspec.GridSpec(2, 3)
gs1.update(wspace=0.2, hspace=0.35) # set the spacing between axes. 

for i in range(0, len(fits)):
    ax = plt.subplot(gs1[i])
```

the you can just loop through the subplots and get the axes

Reply:
Thats a nice solution. Thanks for the hint. 
I choose a different solution: Now the script creates subplots on the fly.